vscode/snippet-generator/pom.xml

```
@@ -28,7 +28,6 @@
     </parent>
 
     <modelVersion>4.0.0</modelVersion>
-    <groupId>org.wso2.ballerina-integrator</groupId>
     <artifactId>ballerina-integrator-snippet-generator</artifactId>
     <version>1.0.0-alpha</version>
```

```suggestion
    <version>1.0.0</version>
```
And please search all the versions in the _pom.xml_ files and correct them.